packages/web3-eth/test/integration/subscribe.test.ts

```diff
@@ -23,26 +23,20 @@ import {
 	SyncingSubscription,
 	NewPendingTransactionsSubscription,
 	LogsSubscription,
-} from '../../src/web3_subscriptions';
+} from '../../src';
 import {
 	getSystemTestProvider,
 	describeIf,
-	getSystemTestAccounts,
 	isWs,
+	createTempAccount,
 } from '../fixtures/system_test_utils';
 
 describeIf(isWs)('subscribe', () => {
 	let web3Eth: Web3Eth;
 	let provider: WebSocketProvider;
-	let accounts: string[];
 
 	beforeAll(async () => {
-		accounts = await getSystemTestAccounts();
```

Comment:
I see, throughout this MR, that `getSystemTestAccounts()` was replaced by  `createTempAccount()` this would lead the tests to take a very long time compared to the old approach. Is there a strong reason for that?
What do you think?